tensorboard/webapp/metrics/store/metrics_store_internal_utils.ts

```diff
@@ -493,9 +504,38 @@ function getNextImageCardStepIndexFromSingleSelection(
   return null;
 }
 
+/**
+ * Gets next stepIndex for an image card based on range selection. Returns null if nothing should change.
+ * @param selectedSteps The selected steps from selected time. It should contain empty or one to multple steps.
```

Comment:
"empty or one to multple steps". That's all possible states of an array so don't know if its just confusing to mention :).

Reply:
That's true. Replacing it with just mention that it could be empty